Server/Auth/ApiAuthorizationFilter.cs

```diff
@@ -25,15 +27,26 @@ public void OnAuthorization(AuthorizationFilterContext context)
 
             if (context.HttpContext.Request.Headers.TryGetValue("Authorization", out var result))
             {
-                var keyId = result.ToString().Split(":")[0]?.Trim();
-                var apiSecret = result.ToString().Split(":")[1]?.Trim();
+                var tokenType = result.ToString().Split(" ")[0].Trim();
 
-                if (DataService.ValidateApiKey(keyId, apiSecret, context.HttpContext.Request.Path, context.HttpContext.Connection.RemoteIpAddress.ToString()))
+                switch (tokenType)
                 {
-                    var orgID = DataService.GetApiKey(keyId)?.OrganizationID;
-                    context.HttpContext.Request.Headers["OrganizationID"] = orgID;
-                    return;
+                    case "Basic":
+                        var encodedToken = result.ToString().Split(" ")[1].Trim();
```

Comment:
This will fail if the auth header does not contain a space.

Additionally, could assign the result of `result.ToString().Split(" ")` to a variable up top to prevent running this logic twice

Reply:
@stevesobol Could I get you to clean this up for me?

Reply:
> Additionally, could assign the result of result.ToString().Split(" ") to a variable up top to prevent running this logic twice

Agreed.  Then fail early if length < 2.

Reply:
@dkattan I'm looking at it now

Reply:
"fail early" implies what, exactly?

This is what I did (and haven't committed yet)
```
                var headerComponents = result.ToString().Split(" ");
                if (headerComponents.Length < 2)
                {
                    context.HttpContext.Response.StatusCode = (int)HttpStatusCode.Forbidden;
                    return;
                };

                var tokenType = headerComponents[0].Trim();
                var encodedToken = headerComponents[1].Trim();
```

Reply:
This looks good to me, Go ahead and commit it.

Reply:
Done